lmfit/model.py

```diff
@@ -993,3 +993,14 @@ def plot(self, datafmt='o', fitfmt='-', initfmt='--', numpoints=None,
                             fit_kws=fit_kws, ax_kws=ax_res_kws)
 
         return fig
+
+    def aic(self):
+        """calculate and return the AIC for a ModelFit.
+        AIC is Akaike information criterion, a measure of the relative quality of a statistical model for a given set of data.
+        It is computed as `n * log(rss/n) + 2k`, where `n` is the number of data points, `rss` is the sum of residuals squares, 
+        and k in the number of varying parameters.
```

Comment:
For your reference, @yoavram, the [numpy doc standard](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt). I don't think the entire lmfit codebase necessarily adheres, but the newer parts (model.py, fitter.py) do, and it's work in progress.